### DISCUSSION TITLE
When passing #50 score in the gameplay leaderboard, the score's place becomes "-"

### DISCUSSION THREAD

**Discussion by Highmagik** [ppy/osu#22041] [Ideas] [answered]
When you pass 50th guy in the leaderboard it then shows his place as "-". I think it's clear that his place becomes 51st, so why not show "#51" instead of "-"
![image](https://user-images.githubusercontent.com/71317439/210881298-304603ee-82c1-4dbc-8e88-8abd62357265.png)

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/22043
